packages/block-library/src/cover/editor.scss

```diff
@@ -116,3 +117,12 @@
 .block-editor-block-patterns-list__list-item .has-parallax.wp-block-cover {
 	background-attachment: scroll;
 }
+
+// Ensure the placeholder container styles are consistent when resizing.
+.wp-block-cover.is-placeholder .cover-block__cover-placeholder-container {
+	background: $white;
+	width: 100%;
+	.block-editor-media-placeholder {
```

Comment:
Tiniest of nits: it looks like the `box-shadow` comes from the `.components-placeholder` class, but the element here also happens to have the `.block-editor-media-placeholder` class name. Would it be worth switching to the other class name for consistency, or is it better to keep it as-is for specificity?

Reply:
This will need massaging as well, as we need the resting and focus states to be the same as other placeholders, which very probably requires we use a box shadow after all. Happy to help if this turns gnarly.

Reply:
Thanks for noticing this! I'll work on it and get back to you if the gnarliness defeats me! :)